deploy-lighthouse-reth.py

```
@@ -0,0 +1,896 @@
+    if BN_ADDRESS:
+        _beaconnodeparameters=f'--beacon-nodes={BN_ADDRESS}'
+    else:
+        _beaconnodeparameters = (
+            f'--beacon-nodes=http://{CL_IP_ADDRESS}:{CL_REST_PORT}'
+            if CL_IP_ADDRESS
+            else '--beacon-nodes=http://127.0.0.1:5052'
+        )
+
+    # Process custom testnet configuration
+    if eth_network=="ephemery":
+        file_path = f"/opt/ethpillar/testnet/bootstrap_nodes.txt"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)